src/components/wallet/WalletImport.tsx

```diff
@@ -53,7 +53,6 @@ function MnemonicField(props: TextFieldProps & MnemonicFieldProps) {
             label={label}
             error={error}
             autoFocus={autoFocus}
```

Comment:
why did you remove default value?

Reply:
This is from the warning it gave me "Input elements must be either controlled or uncontrolled"